app/styles/_common.scss

```diff
@@ -0,0 +1,10 @@
+@mixin vertical-align($position: relative) {
+  position: $position;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+#react-root, .app, body, html {
+  width: 100%;
```

Comment:
well yes the way I've been positioning `<Quiz>` in the page is vertical-aligning it center, which requires width and height 100%... but that layout seems a bit overkill or it'll be annoying when the questions and their answers become longer than the viewport.

i'll swap this out for something simpler.